features/command_line_interface/exit_status.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Return the correct exit status codes
+  In order to use Rubycritic in Continuous Integration environments
+  Rubycritic returns the exit status code properly
+
+  Scenario: Status indicates a success when analyse a clean file
+    Given the clean file 'clean.rb'
+    When I run rubycritic clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a smelly file
+    Given the smelly file 'smelly.rb'
+    When I run rubycritic smelly.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a clean file with the minimum score option
+    Given the clean file 'clean.rb'
+    When I run rubycritic --minimum-score 100 clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates an error when analyse a smelly file with the minimum score option
+    Given the smelly file 'smelly.rb'
```

Comment:
>> That's not the case or this is not a problem?

Actually I think this is good because that you actually realise that something has changed.
Check out [this feature](https://github.com/troessner/reek/blob/master/features/samples.feature) in `Reek`. 
It is huge. We could  easily make that a lot thinner and remove some duplication. We don't. Because if we change something deep down and do NOT expect anything to change in `Reek`s behaviour and all of a sudden we have some changes there we know something went south.
Also, I'd take readability over churn any day, so let's go for readability ;)